photo/admin.py

```diff
@@ -58,11 +58,19 @@ class ContestAdmin(admin.ModelAdmin):
 
     def close_contest(self, request, queryset):
         for contest in queryset:
-            if contest.voting_phase_end > timezone.now():
+            if (
```

Comment:
shouldn't the condition be "if contest.voting_phase_end and contest.voting_phase_end > timezone.now()"?

Reply:
if contest.voting_phase_end is null that expression raises an error because the time comparison must be between two not null dates

Reply:
That is a ternary expression, the comparison would only be executed if the value in the if was true. I have changed it back to an and operator expression